lib/picker.date.js

```diff
@@ -1222,16 +1222,15 @@ DatePicker.prototype.nodes = function( isOpen ) {
 
                                             return klasses.join( ' ' )
                                         })([ settings.klass.day ]),
-                                        'data-pick=' + targetDate.pick + ' ' + _.ariaAttr({
+                                        'data-pick=' + targetDate.pick + ' ' + 'id=' + targetDate.pick + ' tabindex="0" ' + _.ariaAttr({
                                             role: 'gridcell',
                                             label: formattedDate,
                                             selected: isSelected && calendar.$node.val() === formattedDate ? true : null,
-                                            activedescendant: isHighlighted ? true : null,
+                                            activedescendant: isHighlighted ? targetDate.pick : null,
```

Comment:
activedescendent should not be a boolean but the id of the element.
element here is each date of the month

Reply:
Same problem is in picker.time.js as well. However, the supposed fix is being added to the wrong role. 
Reference: https://www.w3.org/TR/wai-aria-1.1
Activedescendant needs to be added to roles grid and combobox for DatePicker and added for roles combobox and listbox for TimePicker.  

Each grid item and list item needs an id attribute set to be passed to aria-activedescendant attribute.